src/pages/signUp/SignUp.tsx

```diff
@@ -1,17 +1,22 @@
 import React, { useState } from "react";
+import { useDispatch } from "react-redux";
 
 import Button from "../../components/Button";
```

Comment:
we discussed it, combine you imports, why you split your component button and form ?